### DISCUSSION TITLE
Welcome to jsonresume-theme-elegant Discussions!

### DISCUSSION THREAD

**Discussion by mudassir0909** [mudassir0909/jsonresume-theme-elegant#124] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by baciucristian** [comment]
Hello
How to export to pdf my resume.json? Even if I install the package it says `You have to install this theme relative to the folder to use it e.g. 'npm install jsonresume-theme-elegant'`

**Reply by mudassir0909** [reply]
```
resume export resume.pdf --theme=elegant
```

does this not work?

**Reply by baciucristian** [reply]
It doesn't work either. And when I export in .html the icons are not showing.
![Screenshot_2](https://user-images.githubusercontent.com/49842769/108039070-4bfeaa00-7044-11eb-8891-20e3a91095c1.png)

**Reply by mudassir0909** [reply]
Umm that is strange, I'll take a look

**Reply by maksymgendin** [reply]
@mudassir0909 I would be also thankfully interested in a correct PDF export. It would be also great to have nicer page margins in the export and to remove the navigation bubble in the bottom right corner...

**Reply by mudassir0909** [reply]
PRs welcome, I also have a dedicated `.less` file to target print versions

https://github.com/mudassir0909/jsonresume-theme-elegant/blob/master/assets/less/print.less

Unfortunately, not getting enough time to spend on this project(story of all the OSS projects innit?). I have a [plan for 2.0](https://github.com/mudassir0909/jsonresume-theme-elegant/pull/127), we can sync up and work together if you want to.

**Reply by maksymgendin** [reply]
I found out why the icons are not loading.

When served through a web server the font paths are resolved properly to e.g. `http://unpkg.com/jsonresume-theme-elegant@1.12.0/assets/icomoon/fonts/icomoon.woff?f218wo`

But when opening the HTML as file in a browser (what the resume-cli does when exporting the PDF), it resolves to
`file://unpkg.com/jsonresume-theme-elegant@1.12.0/assets/icomoon/fonts/icomoon.woff?f218wo` which obviously cannot be loaded.

This happens because in `icon.less` you have `@font-base-path: "//unpkg.com/jsonresume-theme-elegant`. Don't know what is the best way to fix this.

**Reply by maksymgendin** [reply]
I managed to create a proper PDF export via serving the generated HTML file via a Node HTTP server and using [playwright](https://github.com/microsoft/playwright) as headless browser. One thing which I figured out after an hour was that I had to start Chromium with `--disable-web-security` flag so that the icons are displayed properly 😄 Didn't go into details - why.
